.circleci/config.yml

```
@@ -4,6 +4,9 @@ orbs:
   win: circleci/windows@2.2.0
 
 commands:
+  win_install_go:
+    steps:
+      - run: choco install golang --version 1.14.6
```

In case any prompting...
```suggestion
      - run: choco install golang --version 1.14.6 --yes
```